src/components/Person/Person.jsx

```diff
@@ -1 +1,29 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const isMale = sex === 'm';
+  const marriedFemale = !isMale && isMarried;
+  const marriedMale = isMale && isMarried;
+
+  let nameOfPartner;
```

Comment:
Two things here:

1. You modify this variable into a string later, so it's better to declare it as an empty string, to somehow declare a future type of value that will be placed here. 
2.  Not exactly the best naming here. `nameOfPartner` ? Maybe, `partnerStatus` or something like that will do better, because when you see 'name' you probably expect to receive a name here, like 'Magda'.